src/app/src/test/kotlin/org/imperial/mrc/mint/integration/endpoints/ImpactTests.kt

```diff
@@ -31,4 +31,10 @@ class ImpactTests: EndpointTests() {
         val responseEntity = testRestTemplate.getForEntity<String>("/impact/table/config")
         assertSuccess(responseEntity, "TableDefinition")
     }
+
+    @Test
+    fun `can get impact docs`() {
```

Comment:
missing test for cost docs here?

Reply:
that should be in `CostTests` class :)  https://github.com/mrc-ide/mint/pull/66/files#diff-b145b19b1864f9e5b9a7ee9bbd72aa6ca12e19750919ae2df3c83daf79fb14abR28

Reply:
:facepalm: